subprojects/packagefiles/tree-sitter/meson.build

```diff
@@ -0,0 +1,17 @@
+project('tree-sitter', 'c')
+
+tree_sitter_lib = static_library('tree-sitter-runtime', [
+  'lib/src/lib.c',
+  ]
+,
```

Comment:
clean up your white space. Lots of misalignment

Reply:
done